lib/features/details_page/details_page.dart

```diff
@@ -0,0 +1,122 @@
+
+    return (types != null)
+        ? Scaffold(
+            backgroundColor: types?.first.name.toString().pokemonColor,
+            appBar: AppBar(backgroundColor: red),
+            body: Center(
+              child: Column(
+                children: [
+                  Padding(
+                      padding: const EdgeInsets.all(12.0),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            specificPokemon?.name.toString().replaceDash.toTitleCase ?? '',
+                            style: textTheme.headline1,
+                          ),
+                          Align(
+                            alignment: Alignment.centerRight,
+                            child: Text(
+                              specificPokemon?.url?.pokemonId.toString().formatID ?? '',
+                              style: textTheme.headline1?.copyWith(fontSize: 16),
+                            ),
+                          ),
+                          Row(
+                            children: [...?types?.map((type) => Type(typeName: type.name))],
+                          ),
+                          Center(
+                            child: CachedNetworkImage(
+                              imageUrl: specificPokemon?.url?.pokemonId.toString().pokemonImage ?? '',
+                              height: 210,
+                              placeholder: (context, url) => spinKitRipple,
+                              errorWidget: (context, url, error) => const Icon(Icons.error),
+                            ),
+                          ),
+                          SizedBox(height: 10),
+                        ],
+                      )),
+                  Expanded(
+                    child: Container(
+                      decoration: BoxDecoration(
+                        color: white,
+                        borderRadius: BorderRadius.vertical(top: Radius.circular(25)),
+                        boxShadow: [
+                          BoxShadow(
+                            color: Colors.black26.withOpacity(0.2),
+                            blurRadius: 5.0,
+                            spreadRadius: 0.10,
+                          )
+                        ],
+                      ),
+                      child: DefaultTabController(
+                        length: 4,
+                        initialIndex: 0,
+                        child: Column(
+                          mainAxisSize: MainAxisSize.min,
+                          children: [
+                            TabBar(
+                              indicatorColor: red,
+                              labelColor: black,
+                              labelPadding: EdgeInsets.symmetric(horizontal: 10.0),
+                              tabs: [
+                                Tab(text: about),
```

Comment:
You could create a `List<String>` in `string_constants.dart` and `map` it to `Tab` widget